src/cmd2.cc

```diff
@@ -717,41 +657,6 @@ static void chest_death(int y, int x, s16b o_idx)
 
 
 /*
- * Chests have traps too.
```

Comment:
I've seen it at times. Not sure if it applies in the rm-traps branch... Also, chests generated in the wilderness are always empty, FWIW.